lib/components/viewers/stop-schedule-table.js

```diff
@@ -122,16 +122,23 @@ class StopScheduleTable extends Component {
               ? nextStopTime === highlightedStopTime
               : highlightRow
             const routeName = route.shortName ? route.shortName : route.longName
-            const formattedTime = formatSecondsAfterMidnight(stopTime.scheduledDeparture, timeFormat)
 
+            // Convert to milliseconds for FormattedTime
+            const formattedTime = moment().startOf('day').add(stopTime.scheduledDeparture, 's').valueOf()
```

Comment:
Rename, the result is just a timestamp.

Reply:
Renmaed in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/b4b757b5e0da96045794b7a642d7271e79218c10